### ISSUE TITLE
build fails on aarch64

### ISSUE THREAD

**Issue by cappadokes** [google/tcmalloc#121]
CPU: ARMv8 rev 1 (aarch64, Little Endian)
OS: Ubuntu 16.04.7 LTS
Python 3.10.2
gcc 9.4.0
bazel 5.0.0 (installed through Bazelisk)

After cloning `tcmalloc` and running `bazel test //tcmalloc/...` build fails with error:

```bash
ERROR: /SD_32GB/home/cappadokesm/.cache/bazel/_bazel_root/11b7d896c1d7c5b087820c09ea88c09b/external/fuzzing_py_deps/pypi__absl_py/BUILD:5:11: no such package '@fuzzing_py_deps//pypi__enum34': BUILD file not found in directory 'pypi__enum34' of external repository @fuzzing_py_deps. Add a BUILD file to a directory to mark it as a package. and referenced by '@fuzzing_py_deps//pypi__absl_py:pypi__absl_py'
ERROR: Analysis of target '//tcmalloc/internal:numa_fuzz_corpus' failed; build aborted: 
```

**Comment by jwbee** [comment]
This isn't just an aarch64 problem. This fails everywhere. `rules_fuzzing` is just broken. It requires a package that it fails to declare.

**Comment by ckennelly** [comment]
The `rules_fuzzing` issue was resolved by #141.
